Makefile

```diff
@@ -5,5 +5,8 @@ pyclean:
 test: pyclean
 	pytest tests -rs -v --durations=0
 
+test-minimal: pyclean
+	pytest tests -rs -v --durations=0 -m "not slow"
```

Comment:
What about a `MINIMAL=1` option instead of a separate rule? i.e.

```
make test
make test MINIMAL=1
```

(If you think it's a good idea, I can help with setting it up)

Reply:
Could you tell me what is the advantage of using `MINIMAL=1` ? I feel it would make the option less discoverable (since `make` won't list it) and a bit harder to type (no autocompletion).

Reply:
Both your points make a lot of sense! I suggested using a parameter out of personal preference, but after your remarks I agree with you :)

Also, sorry for not writing "nit" in front of my comment!